backend/server/src/db/cmd.rs

```diff
@@ -10,6 +17,11 @@ use super::{Db, create_pool, query};
 
 
 pub(crate) async fn run(cmd: &DbCommand, config: &config::Db) -> Result<()> {
+
+    if let DbCommand::Console = cmd {
+        console(config).map(|_| ())?
+    }
```

Comment:
Having an empty line as the first line within a function is uncommon.

Also: you forgot to add a `return` of some sorts. Or some other way to clearly signal this `if` is never returning.